guides/defensive-domain-registration.md

```diff
@@ -0,0 +1,49 @@
+---
+expires: 2021-06-01
+---
+# Defensive domain registrations
+
+The MOJ and associated organisations (Executive agencies, non-departmental public bodies and so on) main varying levels of 'online presence' using domain registrations that are often a cornerstone of the organisation's identity on the public internet, for example, as the email domain for contacting other government organisations, partners and members of the public.
+
+Each MOJ organisation **must** identify a core set of internet domains it considers critical to it’s internet identity and defensively register a small number of overt variations (for example, `justice.gov.uk` may justify `justicegov.uk`, `justice.co.uk` and `justice.uk`) to protect itself, partners and members of the public from illegitimate parties pretending to be the organisation when they are not (this usually occurs as phishing emails).
+
+## Limiting the permutations to register
+
+Domain permutations for defensive registration should be limited to the organisation's core identity, as opposed to tertiary campaigns/identities, in order to keep costs and management overheads down.
+
+Some domain registrars have programmes to detect malicious registrations of overtly government-associated domains through the use of misspellings and so on, so unless accompanied by strong justifications as to why misspellings must be covered, organisations should only defensively register `.uk` and `.co.uk` top-level domain variants and visual manipulations (for example, the removal of a dot from justice.gov.uk leads to justicegov.uk which would be a registerable domain that looks a lot like justice.gov.uk to the naked eye)
```

Comment:
programmes -> tools
so on, so unless accompanied by strong -> so on. Unless there are strong
manipulations (for example, -> manipulations. For example,
from justice.gov.uk leads to justicegov.uk which would be a registerable domain that looks a lot like justice.gov.uk to the naked eye) -> from `justice.gov.uk` leads to `justicegov.uk` which would be a registerable domain *and* one that looks a lot like `justice.gov.uk` during a casual inspection.